roles/ansible_openwrtnetwork/templates/functions.jinja2

```diff
@@ -370,9 +386,7 @@ config interface "{{ key }}"
 {% endif %}
 {% if value['proto'] == "pppoe" %}
         # PPPoE config
-{% if value['ipv6'] is defined %}
```

Comment:
Why did you reverse that? I don't remember exactly why I built it like that back then but now there is no possibility to set IPv6 attributes any more for PPP interfaces.

Reply:
So I was bit confused about that - the 'ipv6' setting, according to the [openwrt documentation](https://openwrt.org/docs/guide-user/network/network_configuration#common_options) is a common setting that applies to all of the interfaces, not only to the pppoe ones, so that I have [moved it](https://github.com/imp1sh/ansible_openwrt/blob/453a4556e08be072ed45c563ff5985d910271919/roles/ansible_openwrtnetwork/templates/functions.jinja2#L194) to the other section. What am I missing here?

Reply:
Yeah that might be true but in PPP connection I want the default to be IPv6 set to auto. Can you please revert that lines and I will gladly accept your PR.

Reply:
Please have a look at the line 389 - I have not removed the functionality of setting the 'auto', it's still there :) I did not want to introduce any changes in default behaviour.

Reply:
All that has changed is when the `ipv6` options are defined, they are rendered before the `# PPPoE config` as they are likely to be defined for other types of interfaces (i.e. I'm using them on VPN tunnels).

If they are NOT defined, it will still render `option ipv6 "auto"` as it did before.

Reply:
Yeah I see what you did, but I will have to check what this could result in. FYI the OpenWrt documentation is not ALWAYS precise even though it's pretty good.
I will have to make some tests on this. Might take some time though.

Reply:
The `option ipv6 ...` is still printed in the exact same section, so the difference is purely cosmetic, correct?
I am happy to move it back if you think it will have some impact, but honestly I cannot think of any thing that it could impact. Let me know either way